sdk/maps/Azure.Maps.Weather/tests/MapsWeatherTests.cs

```diff
@@ -0,0 +1,309 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using NUnit.Framework;
+using Azure.Core.TestFramework;
+using Azure.Core.GeoJson;
+using Azure.Maps.Weather.Models;
+using Azure.Maps.Weather.Models.Options;
+using System.Runtime.InteropServices;
```

Comment:
Let's put similar namespaces together. Please help to move `System.Runtime.InteropServices` in line 7.